examples/BSTLD.py

```diff
@@ -10,6 +10,8 @@
 # pylint: disable=import-error
 # pylint: disable=pointless-string-statement
 # pylint: disable=invalid-name
```

Comment:
why we have unused import？

Reply:
The unused imports are used to explain how to use regular import when writing own dataloaders.